packages/paste-core/components/combobox/stories/customization.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import {Stack} from '@twilio-paste/stack';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {Card} from '@twilio-paste/card';
+import {Text} from '@twilio-paste/text';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {Theme} from '@twilio-paste/theme';
+import {Combobox} from '../src';
+
+const items = ['a', 'b', 'c'];
+
+interface GroupedItem {
+  letter: string;
+  number: number;
+}
+
+const groupedItems = [
+  {letter: 'a', number: 1},
+  {letter: 'a', number: 2},
+  {letter: 'b', number: 3},
+  {letter: 'b', number: 4},
+  {letter: 'b', number: 5},
+  {letter: 'c', number: 6},
+  {letter: 'd', number: 7},
+];
+
+const ShowCustomization: React.FC<React.ReactNode> = ({children}): React.ReactElement => (
+  <Stack orientation="vertical" spacing="space50">
+    <Card>
+      <Theme.Provider>
```

Comment:
Do we need this here? What is the purpose of using theme provider here? Our stories are configured to be wrapped with a theme provider, so this might be unnecessary.

Reply:
True!